RIOT-OS/RIOT
# boards/slwstk6000b-*: cleanup

### Contribution description

`slwstk6000b-*` uses shell calls to resolve some variables like the `JLINK_DEVICE` and the `CPU_MODEL`, this can be done more efficiently with `MAKE` builtins.

This PR also takes care of addressing the missing `TODO move CPU_MODEL here based on the daughter board`

Some exports are removed as well as some MODULES moved.

### Testing procedure

- There are a few less syscals, #13174 handles the rest.

master
```
BOARD=slwstk6000b-slwrb4150a strace -ff -e trace=execve make --no-print-directory -C examples/hello-world/ info-debug-variable-QUIET 2>&1 | grep 'execve' | wc -l
4912
```
pr
```
BOARD=slwstk6000b-slwrb4150a strace -ff -e trace=execve make --no-print-directory -C examples/hello-world/ info-debug-variable-QUIET 2>&1 | grep 'execve' | wc -l
4882
```

- everything should still works as before so try a couple of applications.

- murdock will check build stuff

### Issues/PRs references

Relates to #10850
Also relates to #9913
This is a first cleanup step, when #12407 is merged and agreed I will move towards a folder based approach.
